Dockerfile.ubuntu.18.04

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2019, Intel Corporation
+#
+# 'recipe' for Docker to build an Debian package
+#
+# Pull base image
+FROM ubuntu:18.04
+Maintainer John.E.Malmberg <John.E.Malmberg@intel.com>
+
+# Install basic tools
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            autoconf bash curl debhelper dh-make dpkg-dev doxygen gcc \
+            git git-buildpackage locales make patch rpm wget
+
+# libfabric
+RUN apt-get install -y \
+            libibverbs-dev libpsm-infinipath1-dev librdmacm-dev
+
+ENV UBUNTU_URL="https://launchpad.net/ubuntu/+archive/primary/+files"
+RUN curl -sS -f -L -O ${UBUNTU_URL}/libpsm2-2_10.3.58-1_amd64.deb ; \
+    curl -sS -f -L -O ${UBUNTU_URL}/libpsm2-dev_10.3.58-1_amd64.deb ; \
+    apt-get install -y \
+    ./libpsm2-2_10.3.58-1_amd64.deb ./libpsm2-dev_10.3.58-1_amd64.deb
```

Comment:
How come these cannot just be `apt-get install`ed?

How is this going to affect going to be able to `apt-get install libfabric1`?

Reply:
I don't think I'm clear which Ubuntu release those packages at: https://launchpad.net/ubuntu/+archive/primary/+files are from.

Reply:
It seems to have packages for all Ubuntu releases in that directory.
I can not access the directory with a web browser.

Reply:
So this makes me uneasy as we have no indication of which Ubuntu release it's built against.

Can we not pull these from a mirror, where we can name the Ubuntu release in the path so that we know with certainty which release it comes from with a hope of having some assurance of compatibility with the release we are pulling them into?

For example, if `libpsm2-2_10.3.58-1_amd64.deb` were rebuilt across 10 (an exaggeration to prove the point) Ubuntu releases (i.e. they didn't upgrade it to a newer version for newer releases), the libraries that it will use will diverge  in compatibility from the release you are aiming to pull it into.

Reply:
No public mirrors.  We would have to setup our own local repository.

For Ubuntu to rebuild an existing package and not change the version string is a violation of their policy.

Adding the 18.10 repositories to the 18.04 container will cause more components to be pulled from 18.10 than we want to.

If need be in the future, we can add rebuilding of these packages on 18.04 locally.